test/integration/agent/route_test.go

```diff
@@ -440,33 +407,23 @@ func TestRouteTablePolicyOnly(t *testing.T) {
 
 	cLink := &netlink.Dummy{}
 	cLink.Name = "containerLink"
-	err = netlink.LinkAdd(cLink)
-	if err == nil {
-		err = netlink.LinkSetUp(cLink)
-	}
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, netlink.LinkAdd(cLink), "error when creating link")
+	assert.NoError(t, netlink.LinkSetUp(cLink), "error when setting-up link")
 
 	_, ipAddr, _ := net.ParseCIDR("10.10.1.1/32")
 	_, hostRt, _ := net.ParseCIDR("10.10.1.2/32")
-	if err := netlink.AddrAdd(cLink, &netlink.Addr{IPNet: ipAddr}); err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, netlink.AddrAdd(cLink, &netlink.Addr{IPNet: ipAddr}), "error when configuring IP on link")
 	rt := &netlink.Route{
 		LinkIndex: cLink.Index,
 		Scope:     netlink.SCOPE_LINK,
 		Dst:       hostRt,
 	}
-	if err := netlink.RouteAdd(rt); err != nil {
-		t.Error(err)
+	if assert.NoError(t, netlink.RouteAdd(rt)) {
+		t.Logf("route %v indx %d, iindx %d added", rt, rt.LinkIndex, rt.ILinkIndex)
```

Comment:
```suggestion
		t.Logf("route %v LinkIndex %d, ILinkIndex %d added", rt, rt.LinkIndex, rt.ILinkIndex)
```
I think variable name in log info should be at least more detailed than variable name in code.